www/index.html

```diff
@@ -5,15 +5,13 @@
     <meta name="viewport" content="initial-scale=1, maximum-scale=1, user-scalable=no, width=device-width">
     <title></title>
 
-    <link href="lib/ionic/css/ionic.css" rel="stylesheet">
```

Comment:
Not sure if we want this to happen or not, think this was from configuring sass?